src/Symfony/Component/PropertyAccess/PropertyAccessor.php

```diff
@@ -63,10 +71,11 @@ class PropertyAccessor implements PropertyAccessorInterface
      * Should not be used by application code. Use
      * {@link PropertyAccess::createPropertyAccessor()} instead.
      */
-    public function __construct($magicCall = false, $throwExceptionOnInvalidIndex = false)
+    public function __construct($magicCall = false, $throwExceptionOnInvalidIndex = false, Cache $cache = null)
```

Comment:
It's not a hard dependency. If you install PropertyAccess without Doctrine cache it will still work (the typehint is not resolved if null is passed).